frontend/src/tests/lib/services/sns.services.spec.ts

```diff
@@ -91,59 +132,53 @@ describe("sns-services", () => {
         },
       ]);
 
-      snsSwapCommitmentsStore.setSwapCommitment({
-        swapCommitment: mockSnsSwapCommitment(principal(0)),
-        certified: true,
-      });
-
-      expect(get(snsSwapCommitmentsStore)).not.toBe(undefined);
+      expect(get(snsSwapCommitmentsStore)).toBeUndefined();
       expect(get(toastsStore)).toEqual([]);
 
       await loadSnsSwapCommitments();
       expect(spy).toBeCalled();
 
-      expect(get(snsSwapCommitmentsStore)).toBe(undefined);
+      expect(get(snsSwapCommitmentsStore)).toEqual([
+        { swapCommitment: commitment2, certified: true },
+      ]);
       expect(get(toastsStore)).toMatchObject([
         {
           level: "error",
-          text: `There was an unexpected error while loading the commitments of all deployed projects. ${errorMessage}`,
+          text: `There was an unexpected error while loading the commitment of the project. ${errorMessage}`,
         },
       ]);
+
+      // loadSnsSwapCommitment logs the error for the certified and uncertified
+      // call but toastsError logs it again for a total of 3 times.
```

Comment:
[non blocking] not sure I fully understand the comment, is it maybe a typo
```suggestion
      // calls but toastsError logs it again for a total of 3 times.
```